### PR TITLE
[releasing] Add a note on the importance of cutting a release every Wednesday.

### PR DESCRIPTION
**Pull request by jverkoey**

### CODE REVIEW

File: contributing/releasing.md
Comment (romoore) [inline contributing/releasing.md]: Since I don't think we require anyone to actually have arrived at work by 10am Eastern Time, should we push this out to 12pm?

Also, can we specify "New York time" or "Eastern Time (US and Canada)"? Choosing Eastern Standard Time may cause confusion during the Daylight Saving Time months.

File: contributing/releasing.md
Comment (romoore) [inline contributing/releasing.md]: This seems more like a reason for a Hotfix release and isn't really part of the release process itself. Do we have another section where we explain **why** we might cut a hotfix release?

File: contributing/releasing.md
Comment (jverkoey) [reply contributing/releasing.md]: This isn't an implication of a requirement that anyone shows up before 10am - it's fair and reasonable to cut the release Tuesday evening prior to leaving work. The important fact here is that we do not let the cutting of the release slip due to waiting for a feature to land.

File: contributing/releasing.md
Comment (jverkoey) [reply contributing/releasing.md]: It bears emphasis because I think it may not have been clear in the past that it's more important for a release to be cut and landed than for any individual feature to make it into a given weekly release.

Comment (jverkoey) [reply contributing/releasing.md]: Are you referring specifically to "If your PR is important, cut a hotfix release in addition to the typical weekly..." ? And your proposal being that we move that sentence to the content regarding hotfixing?

Comment (romoore) [reply contributing/releasing.md]: I am referring to that sentence, yes. I think some general guidance on "when should it be a Hotfix?" would be helpful in a different part of our docs.

File: contributing/releasing.md
Comment (romoore) [reply contributing/releasing.md]: Is that fair to authors of PRs who are trying to "make the deadline"? If the release cop decides to cut the release at 3:00pm on a Tuesday, others hoping to land a PR around 5pm would have to wait one more week.  This is assuming it's not a critical issue warranting a Hotfix release.

Comment (jverkoey) [reply contributing/releasing.md]: Likely not fair, no, but I think this will have to come down to collaboration with the release engineer for a given week. The harsh truth is that if it doesn’t make the cut, it doesn’t make the cut. This fact is in fairness to all the other changes that did make it into the release.

File: contributing/releasing.md
Comment (jverkoey) [reply contributing/releasing.md]: I’ll add a note here linking to the hotfixing docs and move this sentence there.

**Comment by ianegordon** [comment]
I disagree with "without exception"
Historically we have delayed a release to land an important client fix.
We have also intentionally skipped releases due to limited change or people being OOO. 
Google I/O comes to mind.

**Review by ianegordon — DISMISSED** [review]
I think we need flexibility in our release schedule.
The release-cop should be striving for a release ~middle of every week.  I advocate that the decision on when to cut should be driven by client needs, not time of day.

File: contributing/releasing.md
Comment (ajsecord) [reply contributing/releasing.md]: I think this thread isn't worth the engineering time—too small to matter.

I guess one could say "first thing Wednesday morning, NYC time" if you wanted to avoid encoding a specific time, but I defer to Jeff here.

The idea with a regular, predictable release cut is that features that miss one release will just roll into the next—you don't have to wait a long time.

**Review by ajsecord — APPROVED** [review]

**Comment by jverkoey** [comment]
> Historically we have delayed a release to land an important client fix.

The intent of this change is to encourage a healthy understanding of the fact that we can always land something that's critical in a follow-up off-schedule release. The scheduled releases should never be gated on any functionality landing at the last minute.

> We have also intentionally skipped releases due to limited change or people being OOO. Google I/O comes to mind.

I think that in these cases we should still be cutting releases. Even if they are just hotfix releases.

**Review by ianegordon — APPROVED** [review]
Discussed offline

**Pull request merged**
